submissions/sprint_a/readme.txt

```diff
@@ -0,0 +1,5 @@
+This program can generate a random epithet for you upon request. 
+It takes a given json file or csv and will use the vocabulary in
+those files to generate said epithet. You can request one or multiple
+epithets. You can also request the vocabulary by itself to see 
+what vocabulary the program pulls from.
```

Comment:
readmne files are CAPITALIZED by convention and Python projects tend to use reStructured Text or Markdown formats for READMEs. Markdown is more popular but reStructured Text is used to document Python projects and the language itself.

- [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
- [reStructed Text](https://github.com/ralsina/rst-cheatsheet/blob/master/rst-cheatsheet.rst)